sysutils/cpmtools/Portfile

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem		1.0
+
+name			cpmtools
+version			2.20
+categories		sysutils
+platforms		darwin
+license                 GPL-3
+maintainers		@codesmythe openmaintainer
```

Comment:
You'll want to also include your obscured email address, as in:

```
maintainers		{domain.com:user @codesmythe} openmaintainer
```

Reply:
Thanks for the review. I have updated my email address in the Portfile.